kicad_AmigaAdapter/VideoSlot/V2/gerber/videoslotadapter-F_SilkS.gbr

```diff
@@ -0,0 +1,2183 @@
+G04 #@! TF.GenerationSoftware,KiCad,Pcbnew,5.1.9-73d0e3b20d~88~ubuntu20.10.1*
```

Comment:
If this silk screen includes "Modified by LinuxJedi" I would suggest sticking with the precedent that c0pperdragon set and I also followed for the V1 of the video slot adapter, of leaving credits to a single person off the silk screen.

Reply:
Fair enough, I'll fix that up shortly.